### PR TITLE
GraphQL - query site links

### PR DESCRIPTION
**Pull request by toyhammered**
# What

Add site links to graphql

# Checklist

- [ ] All files pass Rubocop
- [ ] Any complex logic is commented
- [ ] Any new systems have thorough documentation
- [ ] Any user-facing changes are behind a feature flag (or: explain why they can't be)

- [ ] All the tests pass
- [ ] Tests have been added to cover the new code

### CODE REVIEW

**Comment by toyhammered** [comment]
@wopian Can you help with the descriptions, I am not really sure the best way to describe them.

File: app/graphql/types/profile.rb [hunk 1]
```diff
@@ -44,6 +44,14 @@ class Types::Profile < Types::BaseObject
     null: true,
     description: 'The message this user has submitted for the Hall of Fame'
 
+  field :site_links, Types::SiteLink.connection_type,
+    null: true,
+    description: 'Links related to a profile.'
```
Comment (wopian) [inline app/graphql/types/profile.rb]: 1. Links to the user on other (social media) sites

2. The user's accounts on other (social media) sites

3. Links to the user on external sites

File: app/graphql/types/site_link.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class Types::SiteLink < Types::BaseObject
+  description 'A link to an external site related to a specific user profile'
```
Comment (wopian) [inline app/graphql/types/site_link.rb]: A link to a user's profile on an external site?

File: app/graphql/types/site_link.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+  field :id, ID, null: false
+
+  field :url, String,
+    null: false,
+    description: 'The full url for the site, including the user profile.'
```
Comment (wopian) [inline app/graphql/types/site_link.rb]: A fully qualified URL of the user profile on an external site?

**Review by wopian — COMMENTED** [review]
.

**Pull request merged**